python/sparkdl/graph/builder.py

```diff
@@ -207,7 +207,9 @@ def fromList(cls, functions):
         The inputs and outputs are picked out of the scopes, so that users
         will still be able to call the function with the expected inputs/outputs names.
 
-        It is assumed that there is only one input and one output in the intermediary layers
+        It's assumed that the input/output names of adjacent GraphFunction elements in
```

Comment:
AFAICT, the updated comment here reflects the function (`GraphFunction.fromList`)'s actual behavior. I'm thinking we should test that this function works when gluing together graphs with > 1 input/output tensor.

IMO the easiest (least code) way to do so would be to add a TFTransformer test with multiple inputs/outputs; the test could also have input types other than tf.float64 (to verify that we now handle other input types).

However I'm also open to creating separate unit tests for this function (verifying gluing together of multiple inputs/outputs) and TFTransformer (verifying that we can handle non-float64 inputs). Open to suggestions :)

Reply:
Let's add a few tests with very simple graphs for this function specifically. It'll be easier to reason about and detect problems. Thanks sid!